common/internal_model/src/main/scala/weco/catalogue/internal_model/work/InvisibilityReason.scala

```diff
@@ -9,4 +9,5 @@ object InvisibilityReason {
   case object UnlinkedHistoricalLibraryMiro extends InvisibilityReason
   case class UnableToTransform(message: String) extends InvisibilityReason
   case object MetsWorksAreNotVisible extends InvisibilityReason
+  case object TeiWorksAreNotVisible extends InvisibilityReason
```

Comment:
nit: possibly rename this to emphasise that it's a temporary condition?

e.g. `TeiWorksAreInDevelopment`